File: backend/src/main/java/com/ameri/objects/beans/adminBeans/AdminBeans.java
package com.ameri.objects.beans.adminBeans;

import com.ameri.objects.classes.user.editor.Comment;
import com.ameri.objects.classes.user.editor.Subscription;

import java.util.List;

public class AdminBeans {

    private int count;
    private int magazineRecord;
    private List<Comment> commentList;
    private List<Subscription> subscriptionList;

    public AdminBeans(int count, int magazineRecord) {
        this.count = count;
        this.magazineRecord = magazineRecord;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public int getMagazineRecord() {
        return magazineRecord;
    }

    public void setMagazineRecord(int magazineRecord) {
        this.magazineRecord = magazineRecord;
    }

    public List<Comment> getCommentList() {
        return commentList;
    }

    public void setCommentList(List<Comment> commentList) {
        this.commentList = commentList;
    }

    public List<Subscription> getSubscriptionList() {
        return subscriptionList;
    }

    public void setSubscriptionList(List<Subscription> subscriptionList) {
        this.subscriptionList = subscriptionList;
    }
}

File: frontend/src/objects/classes/beans/adminBeans/AdminBeans.ts
import {Comment} from "../../usuario/editor/Comment";
import {Subscription} from "../../usuario/editor/Subscription";

export class AdminBeans{

  count: number;
  magazineRecord: number;
  commentList: Array<Comment>;
  subscriptionList: Array<Subscription>;


  constructor(count: number, magazineRecord: number, commentList: Array<Comment>, subscriptionList: Array<Subscription>) {
    this.count = count;
    this.magazineRecord = magazineRecord;
    this.commentList = commentList;
    this.subscriptionList = subscriptionList;
  }
}
